internal/ptr_64.go

```diff
@@ -1,5 +1,5 @@
-// +build !386,!amd64p32,!arm,!mipsle,!mips64p32le
-// +build !armbe,!mips,!mips64p32
+//go:build !386 && !amd64p32 && !arm && !mipsle && !mips64p32le && !armbe && !mips && !mips64p32
```

Comment:
I've noticed fmt understands arch exclusions and added a few of its own. Wouldn't it be more concise to turn this into an allow list now?

Reply:
I don't think `gofmt` added any new arch exclustions, it just re-arranged the existing ones differently.

From my perspective it'd make sense to turn this into an allow list, as it would also mean we have to explicitly update the list whenever a new architecture is added to Go or gccgo (and ideally verify that `cilium/ebpf` works on that architecture).

/cc @lmb who initially added that code

Reply:
FWIW, this would be the current list of 64-bit architectures for the allow list:

```go
// +build alpha amd64 arm64 arm64be mips64 mips64le ppc64 ppc64le riscv64 s390x sparc64
```

Reply:
Alright, that doesn't look much better. Looks like there was a proposal (https://github.com/golang/go/issues/33388) for 32/64-bit build tags, but that didn't make it. Let's leave this as-is.